internal/venice-common/src/test/java/com/linkedin/venice/pubsub/PubSubTopicPartitionInfoTest.java

```diff
@@ -36,4 +40,31 @@ public void testToString() {
     PubSubTopicPartitionInfo partitionInfo = new PubSubTopicPartitionInfo(new PubSubTopicImpl("testTopic"), 0, true);
     assertEquals(partitionInfo.toString(), "Partition(topic = testTopic, partition=0, hasInSyncReplica = true)");
   }
+
+  @Test
+  public void testTopicDeletionPriority() {
+    PriorityQueue<PubSubTopic> allTopics = new PriorityQueue<>(TopicCleanupService.topicPriorityComparator);
+    int numberOfRTTopicsInTest = 4;
+    PubSubTopic topic1 = new PubSubTopicImpl("topic1" + Version.REAL_TIME_TOPIC_SUFFIX);
+    PubSubTopic topic2 = new PubSubTopicImpl("topic2" + Version.REAL_TIME_TOPIC_SUFFIX);
+    PubSubTopic topic3 = new PubSubTopicImpl("topic3" + Version.REAL_TIME_TOPIC_SUFFIX);
+    PubSubTopic topic4 = new PubSubTopicImpl("topic4");
+    PubSubTopic topic5 = new PubSubTopicImpl("topic5");
+    PubSubTopic topic6 = new PubSubTopicImpl("topic6" + Version.REAL_TIME_TOPIC_SUFFIX);
+    allTopics.add(topic1);
+    allTopics.add(topic2);
+    allTopics.add(topic3);
+    allTopics.add(topic4);
+    allTopics.add(topic5);
+    allTopics.add(topic6);
```

Comment:
Is it possible to add some `randomness` into this testing? For example, let's say, in a for loop, we generate real time and version topics based on probability, and count their appearances respectively. In the verification phase, we poll and assert them.